Sources/SwiftSyntaxBuilderGeneration/SyntaxUtilities.swift

```diff
@@ -82,3 +82,30 @@ func createFormatLeadingTriviaParameters(withDefaultTrivia: Bool = false) -> Par
     ]
   )
 }
+
+/// Generate the `create...` function for an `ExpressibleAs...` conformance.
+func createExpressibleAsCreateFunction(type: SyntaxBuildableType, additionalDocComments: [String] = []) -> FunctionDecl {
+  FunctionDecl(
+    leadingTrivia: ([
+      "/// Conformance to `\(type.expressibleAsBaseName)`.",
+    ] + additionalDocComments).map { .docLineComment($0) + .newline }.reduce([], +),
+    modifiers: [TokenSyntax.public],
+    identifier: .identifier("create\(type.buildableBaseName)"),
+    signature: FunctionSignature(
+      input: ParameterClause(),
+      output: type.buildable
+    )
+  ) {
+    ReturnStmt(expression: "self")
+  }
+}
+
+/// Generate the `create...` function for an `ExpressibleAs...` conformance
+/// that includes an explanation as to how the function disambiguates a conformance.
+func createDisambiguatingExpressibleAsCreateFunction(type: SyntaxBuildableType, baseType: SyntaxBuildableType) -> FunctionDecl {
+  createExpressibleAsCreateFunction(type: baseType, additionalDocComments: [
+    "/// `\(type.buildableBaseName)` may conform to `\(baseType.expressibleAsBaseName)` via different `ExpressibleAs*` paths.",
+    "/// Thus, there are multiple default implementations of `create\(baseType.buildableBaseName)`, some of which perform conversions",
+    "/// through `ExpressibleAs*` protocols. To resolve the ambiguity, provie a fixed implementation that doesn't perform any conversions.",
+  ]) 
+}
```

Comment:
This PR factors out the `create...` functions generated for the `ExpressibleAs...` conformances. This is particularly nice since we can share a lot of logic between `BuildableNodes` and `BuildableCollectionNodes`, including this lengthy doc comment that explains how the function disambiguates the implementation.